libsolidity/codegen/CompilerContext.cpp

```diff
@@ -422,7 +423,7 @@ void CompilerContext::appendInlineAssembly(
 			BOOST_THROW_EXCEPTION(
 				StackTooDeepError() <<
 				errinfo_sourceLocation(nativeLocationOf(_identifier)) <<
-				util::errinfo_comment("Stack too deep (" + to_string(stackDiff) + "), try removing local variables.")
```

Comment:
If this is always using the same text, does it make sense to add the comment in the constructor of the error?

Reply:
What do you mean?